components/filter-components/MapFilters.vue

```diff
@@ -183,13 +183,16 @@
                       {{ propertyItem.label }}
                     </p>
                     <div
-                      class="inline-flex items-center justify-center align-middle rounded-full border-r-2 border-l-2 border-gray-400 w-5 h-5 cursor-pointer"
+                      class="inline-flex items-center justify-center align-middle w-5 h-5 ml-1 mb-1 cursor-pointer"
                     >
                       <img
-                        src="/images/infoIcon.svg"
-                        alt="Filter Information Popup"
+                        src="/images/questionIcon.svg"
+                        aria-label="Filter Description Tooltip"
                         class="pt-0.5 h-5 inline"
                       />
+                      <span class="visually-hidden"
+                        >Filter Description Pop Up</span
```

Comment:
We should keep this the same as the aria-label. So just "Filter description" should be fine. Thanks

Reply:
done

Reply:
Thank you!